tasks/modules.py

```diff
@@ -114,6 +114,17 @@ def dependency_path(self, agent_version):
     "pkg/remoteconfig/state": GoModule("pkg/remoteconfig/state"),
 }
 
+"""
+These sub-modules are not allowed to import the datadog-agent repository, in order to retain their independence
+and facilitate their external usage.
+"""
+INDEPENDENT_MODULES = {
+    "pkg/quantile": GoModule("pkg/quantile"),
+    "pkg/obfuscate": GoModule("pkg/obfuscate"),
+    "pkg/trace": GoModule("pkg/trace"),
+    "pkg/otlp/model": GoModule("pkg/otlp/model"),
```

Comment:
Can you add `"pkg/security/secl": GoModule("pkg/security/secl"),` as well ? For the exact same reasons

Reply:
Sure thing! Wasn't sure if it should be there. Done!

Reply:
Thanks !